homeassistant/components/fritzbox/switch.py

```diff
@@ -92,11 +87,6 @@ def extra_state_attributes(self) -> SwitchExtraAttributes:
             ATTR_STATE_LOCKED: self.device.lock,
         }
 
-        if self.device.has_powermeter:
-            attrs[
-                ATTR_TOTAL_CONSUMPTION
-            ] = f"{((self.device.energy or 0.0) / 1000):.3f}"
-            attrs[ATTR_TOTAL_CONSUMPTION_UNIT] = ATTR_TOTAL_CONSUMPTION_UNIT_VALUE
         if self.device.has_temperature_sensor:
             attrs[ATTR_TEMPERATURE] = str(
```

Comment:
Would be nice to move the temperature sensor too. We don't any extra attributes to entities nowadays that are valid as separate sensors. This can also be done in a separate PR.

Reply:
this is already done with #52558, but honestly forgot to remove the extra attribute :see_no_evil: